### DISCUSSION TITLE
How do I generate strings for the different languages?

### DISCUSSION THREAD

**Discussion by bunabyt3** [rxhanson/Rectangle#1122] [Q&A]
As part of the contribution I'm currently working on, I need to add a new option to the settings window. I'm aware I will need to translate this in to multiple languages. How do I create a string in all of the supported languages for this checkbox label? Is there a way to do it automatically?

**Comment by rxhanson** [comment]
I'm not aware of any way to do it automatically. I generate a new file for a language that doesn't exist, and copy paste it into all of them, then delete the new file. If you figure out a better way, I'd love to hear it.
